docs/index.rst

```diff
@@ -137,13 +141,14 @@ Some of our favourite tools include:
 .. _Authenticate: https://crate.io/docs/crate/reference/en/latest/admin/auth/index.html
 .. _blog post: https://crate.io/a/visualize-crate-data-with-metabase/
 .. _client compatibility notes: https://crate.io/docs/crate/reference/en/latest/interfaces/postgres.html#client-compatibility
-.. _crate_ruby: https://rubygems.org/gems/crate_ruby
+.. _Crate.Client: https://github.com/mfussenegger/crate-mono
```

Comment:
Why are we adding this link? It doesn't seem to be used in the doc and the project appears to be archived as well.

Reply:
I ran a sort on the whole list and so it got moved from https://github.com/crate/crate-clients-tools/pull/5/files#diff-caf2a6b8f4947d018f68893c695b5202L145 

Thank you for catching that it isn't referenced anymore. I'll remove it.